src/node/internal/internal_comparisons.ts

```diff
@@ -0,0 +1,560 @@
+
```

Comment:
Is this our own code?

Reply:
oh, right, forgot to add the header. good catch